packages/FormElement/src/components/Help/Help.js

```diff
@@ -28,6 +27,9 @@ function Help(props) {
 }
 
 const propTypes = {
+  /** zIndex when component using in modals */
```

Comment:
I wouldn't mention the modal use case – there could be many different reasons to use this new prop.